skipruntime-ts/examples/database.ts

```diff
@@ -79,9 +78,7 @@ const data: Entry<string, User>[] = db
 
 db.close();
 
-const instance = await initService(serviceWithInitialData(data));
-
-const closable = runService(instance, {
+const closable = await runService(serviceWithInitialData(data), {
```

Comment:
While you are here, note that at some point `runService` got a return type named `SkipServer`, so renaming `closable` to `server` might make sense. And similarly for the other examples.